src/main/resources/templates/admin/shapefile.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="/admin/header :: head"/>
+<style>
+    table.bordered, table.bordered tr th, table.bordered tr td {
+        border: 1px solid black;
+    }
+
+    table.bordered td, table.bordered th {
+        padding: .25em;
+    }
+
+    .map {
+        width: 100%;
+        height: 400px;
+    }
+</style>
+<body>
+
+<span th:include="/admin/header :: top"/>
+
+<a th:href="|${prefix}/|">Home</a>
+
+<h2>Shapefile viewer</h2>
+
+<!-- TODO: Permission for this -->
```

Comment:
does the canCreateFacility gating satisfy this TODO?

Reply:
Oops, forgot to update this. Thanks.